doc/doxygen/src/ForManagers.dox

```diff
@@ -117,7 +117,7 @@
  *
  * - Implement the methods grouped under the "Entity Reference Inspection"
  *   and "Entity Reference Resolution" sections of the
- *   @ref openassetio.managerApi.ManagerInterface.ManagerInterface
+ *   @fqref{managerApi.ManagerInterface}.ManagerInterface
```

Comment:
[fix] This links to the `ManagerInterface` constructor, probably not what was intended.

Reply:
https://github.com/OpenAssetIO/OpenAssetIO/pull/1058/commits/948efe5518f4da207ec3438f17531c456b524965